.ci/run.py

```diff
@@ -66,6 +66,9 @@ def get_examples_to_skip(current_version):
     if platform.system() == "Darwin":
         skip.extend(['./features/multi_config', ]) # FIXME: it fails randomly, need to investigate
 
+    skip.extend(['./features/emscripten', ]) # FIXME, TODO: Update emscripten example
+    skip.extend(['./libraries/protobuf', ]) # FIXME, TODO: Update protobuf example
```

Comment:
This examples should be updated to support the current conan center index versions, they are still with the deprecated repo ones.